package.json

```diff
@@ -8,7 +8,7 @@
     "fix": "prettier --write .",
     "test": "jest",
     "start": "node server.js",
-    "dev": "local-ssl-proxy --source 443 --target 5500 -n dev.realdevsquad.com"
+    "dev": "node server.js & local-ssl-proxy --source 443 --target 8000 -n dev.realdevsquad.com"
```

Comment:
Thanks for working on this. 

Suggestion: Can we use `node server.js && local-ssl-proxy --source 443 --target 8000 -n dev.realdevsquad.com` command instead? 

adding two `&&` will ensure local-ssl-proxy runs only if node server.js starts successfully this will avoid the issues where the proxy starts but the server fails. I think it will be a more reliable

Reply:
Its basically to run the server in background, if we use `&` with any command, its basically to create a background process @MehulKChaudhari .